src/CattyTests/Bricks/GoToBrickTests.swift

```diff
@@ -0,0 +1,113 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class GoToBrickTests: AbstractBrickTest {
+
+    var brick: GoToBrick!
+    var spriteNode: CBSpriteNode!
+    var object: SpriteObject!
+    var script: WhenScript!
+
+    override func setUp() {
+        super.setUp()
+
+        object = SpriteObject()
+        spriteNode = CBSpriteNode(spriteObject: object)
+        object.spriteNode = spriteNode
+
+        self.scene.addChild(spriteNode)
+        spriteNode.catrobatPosition = CBPosition(x: 0, y: 0)
+
+        script = WhenScript()
+        script.object = object
+
+        brick = GoToBrick()
+        brick.script = script
+    }
+
+    func testGoToBrickTouchPosition() {
+        brick.spinnerSelection = 80
+
+        let action = brick.actionBlock(self.formulaInterpreter.touchManager)
```

Comment:
Please use a test double instead (`TouchManagerMock`) and directly set the last touch position.